Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -118,14 +106,20 @@ class TopSiteItemCell: UICollectionViewCell {
         }
     }
 
-    func configureWithTopSiteItem(site: TopSiteItem) {
-        titleLabel.text = site.urlTitle
-        guard let favURL = site.faviconURL else {
-            contentView.backgroundColor = UIColor.lightGrayColor()
-            imageView.image = FaviconFetcher.getDefaultFavicon(site.siteURL)
-            return
+    func configureWithTopSiteItem(site: Site) {
+        titleLabel.text = site.tileURL.extractDomainName()
+        if let suggestedSite = site as? SuggestedSite {
```

Comment:
I originally wanted the color of the default sites to be in the image itself. But bbell convinced it shouldn't be. So now I check if it's a suggestedSite. If it is I look for the color info from the object itself.  The new images I added are color but I'll be replacing them with white ones as soon as I get the new defaults from bbell.